Makefile

```diff
@@ -24,16 +28,33 @@ pod-update: pod-repo-update
 open:
 	open $(PROJECT_NAME).xcworkspace
 
-test:
+unit-test-ios:
 	@echo "######################################################################"
 	@echo "### Unit Testing iOS"
 	@echo "######################################################################"
```

Comment:
This step runs both unit & functional tests, so I suggest we rename to `test-ios` / `test-tvos`, "Testing iOS" / "Testing tvOS"
Make sure you also update circleci config after that